OpenRA.Game/Traits/RevealsShroud.cs

```diff
@@ -15,13 +15,17 @@ namespace OpenRA.Traits
 	public class RevealsShroudInfo : ITraitInfo
 	{
 		public readonly WRange Range = WRange.Zero;
+		[Desc("Minimum delay in ticks before next visibility update")]
+		public readonly int VisibilityUpdateDelay = 1;
```

Comment:
I think `UpdateDelay` works well enough on its own.
But this is just nit-picking and doesn't change the implementation.